src/main/frontend/components/block.css

```
@@ -657,4 +657,61 @@ html.is-mac {
       cursor: pointer;
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Fix broken styles of breadcrumb caused by mixed contents
+ * TODO: implement the logic in code instead of CSS selector
+*/
+
+/*Fix images*/
+.breadcrumb .image-resize {
+  width: auto;
+  height: 17px;
+  display: inline-block;
+  vertical-align: middle;
+  top: -1px;
+}
+
+.breadcrumb .image-resize img {
+  height: 17px;
+  width: auto;
+}
+
+.breadcrumb .image-resize :is(.asset-action-bar, .erd_scroll_detection_container) {
+  display: none;
+}
+
+/*Fix iframe(e.g YouTube)*/
+.breadcrumb :is(.embed-block>div, iframe) {
+  display: none;
+}
+
+/*Fix embed-block, properties*/
+.breadcrumb>a>div:not([style="display: inline;"]),
+.breadcrumb .embed-block {
+  display: inline-block;
+}
+
+/*Use ... to replace the invalid items*/
+.breadcrumb .embed-block {
+  background: initial;
+}
+
+/*Fix blockquote*/
+.breadcrumb :is(cp__fenced-code-block, blockquote) {
```

```suggestion
.breadcrumb :is(.cp__fenced-code-block, blockquote) {
```
Was that the intention here?

---

> Was that the intended selector here?

yes！